pyproject.toml

```diff
@@ -47,7 +47,7 @@ serial = [
     "pyserial>=3.5"
 ]
 repl = [
-   "pymodbus-repl>=2.0.3"
+   "pymodbus-repl>=2.0.4"
```

Comment:
Seems you did not release that version.

Reply:
It has now been released on [PyPI](https://pypi.org/project/pymodbus-repl/2.0.4/)